twilight-gateway-queue/tests/in_memory.rs

```diff
@@ -0,0 +1,58 @@
+mod common;
+
+use common::{different_id_is_parallel, multi_bucket, reset_after_refills, same_id_is_serial};
+use std::{
+    future::{poll_fn, Future},
+    pin::Pin,
+};
+use tokio::time::{advance, Duration, Instant};
+use twilight_gateway_queue::{InMemoryQueue, Queue, IDENTIFY_INTERVAL};
+
+#[tokio::test]
+async fn disabled_is_instant() {
+    let queue = InMemoryQueue::new(0, 1000, Duration::from_secs(60 * 60 * 24), 1000);
```

Comment:
shouldn't we just use `InMemoryQueue::default` to create these?

Reply:
These are integration tests, so they should only use public API and the `InMemoryQueue::default` implementation should match whatever Discord *currently* uses, i.e. its exact values should not be semver guaranteed. I will highlight this in the `default` implementation documentation.

Reply:
That makes sense